frontend/packages/ceph-storage-plugin/src/components/volume-snapshot/volume-snapshot.tsx

```diff
@@ -0,0 +1,233 @@
+import * as React from 'react';
```

Comment:
Can we have a single default export from this file? And not export things that don't need to be exported.

Reply:
Some people add `export` by default :smiley: but I agree with @bipuladh - I think we should only export things we actually need to import in other modules.

I recall DevConsole plugin following a convention where each React component (file) has a single default export. Other plugins may provide a named, non-default export for the main component. Ideally, each plugin would choose a specific style and follow it consistently :smiley: